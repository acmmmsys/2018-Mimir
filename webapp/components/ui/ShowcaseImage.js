import React, { Component } from 'react';

import 'style/ui/ShowcaseImage.scss'

class ShowcaseImage extends Component {
    render() {
        return (
            <img className='selected-image'
                 src={ this.props.source }
            />
        );
    }
}

export default ShowcaseImage;