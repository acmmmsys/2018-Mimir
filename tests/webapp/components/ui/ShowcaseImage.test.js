import React from 'react';
import ShowcaseImage from '../../../../webapp/components/ui/ShowcaseImage'
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store'
 
const mockStore = configureStore();

let store;
let wrapper;

test('ShowcaseImage does render', () => {
  const wrapper = shallow(
      <ShowcaseImage store={mockStore(initialState)}>This is a test!</ShowcaseImage>
  );
  expect(wrapper).toMatchSnapshot();
});
