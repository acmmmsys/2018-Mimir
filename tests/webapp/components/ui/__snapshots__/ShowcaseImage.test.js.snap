// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowcaseImage does render 1`] = `
<img
  className="selected-image"
/>
`;
